packages/functional/src/__tests__/pipe-transaction-test.ts

```diff
@@ -0,0 +1,215 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+import {
+    BaseTransaction,
+    Blockhash,
+    createTransaction,
+    setTransactionFeePayer,
+    setTransactionLifetimeUsingBlockhash,
+} from '@solana/transactions';
```

Comment:
I probably should have said something about this earlier 😓 but some thoughts now on this test.

1. The unit test doesn't have to be – and probably shouldn't – be specific to transactions. You probably only need to test:
    * What's the return value when you _only_ supply an initial value
    * What's the return value when you supply at least one function that operates over the previous value
    * What happens when one of the transformation functions throws
2. The _more_ interesting test, probably, is a type test.
    * If you only supply an initial value, does the final value match the initial value's type?
    * If you transform a previous value using a function with generics, does the previous type properly flow through the generic?

Like, this is a type test that you'd want to make sure passes.

```ts
function asyncify<TValue>(value: TValue): Promise<TValue> {
  return value;
}
pipe(123, asyncify) satisfies Promise<number>;
pipe('123', asyncify) satisfies Promise<string>;
pipe('123', () => {}, asyncify) satisfies Promise<void>;
```

In any case, these test functions don't have to – and probably shouldn't – come from `@solana/transactions`. Imagine: a breaking change to `@solana/transactions` will break these tests – for no good reason.

Reply:
All fair points, I'll cut `@solana/transactions` out of the lib & tests